### DISCUSSION TITLE
YOLO

### DISCUSSION THREAD

**Discussion by libsource** [community/community#19490] [Other Feature Feedback, Questions, & Ideas]

**Comment by rilysh** [comment]
@libsource To get YOLO badge, you need to merge a PR without reviewing it.

Simply means: whenever you'll get a PR, simply click merge, don't click on "add review".

You can check it out here, https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

Hope this helps.

**Comment by res138** [comment]


**Reply by imran-baitham** [reply]
nice

**Reply by imran-baitham** [reply]
Great it's help full

**Reply by Igorcbraz** [reply]
thank

**Reply by melaxman** [reply]
nicw

**Comment by Nik-08** [comment]
1 Open your repository;
2 Create new branch;
3 Do some updates;
4 Then, create pull request to main branch;
5 IMPORTANT! Add reviewer. To do this, you need to add access to repository someone, then add him as reviewer;
6 After this, just click merge as you do it always;
7 Check your profile and see YOLO achievement.

**Comment by Pro-Bandey** [comment]
To Get YOLO Badge Follow Below Given Steps

Go to your profile for the badge you want to get. 

• Open your repository any one like "github.com/demo/demo"

• Edit any file or  Readme.md file
• save the file "save commit"

• below the commit Check the new branch circle

• in input bar will you see typed like "demo's patch-1"

• Save the commit 

• automatically pull request section open 

• Then, create pull request to main branch;

•  IMPORTANT! Add reviewer. To do this, you need to add access to repository someone, then add him as reviewer

• After this, just click merge as you do it always;

• go to profile and refresh

•  you'll see YOLO achievement.

**Comment by Prince-Rim** [comment]
Don't you need a collaborator to do this?

**Reply by Pro-Bandey** [reply]
Yes Don't need a collaborator

**Comment by melaxman** [comment]
nice

**Comment by Hilal-11** [comment]
Here's a short answer for the YOLO discussion:

YOLO is earned by merging a pull request without requiring a review — i.e. you open a PR, request a reviewer, but merge it yourself before they review/approve.

Add a collaborator to your repo (you need someone to request as reviewer — can't assign yourself).
Create a branch, make a small change, commit, open a PR.
Request that collaborator as a reviewer.
Don't wait — click Merge pull request yourself before they review.
